From cebd3153eba86914934c598e0c617e16e50cd329 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Tue, 7 Mar 2023 10:51:56 +0100
Subject: [PATCH 287/841] docs(changelog): reorder ST drivers scopes

Reorder the ST drivers scopes alphabetically.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 30435589d62a37a6eed6d8f0600a6d8441a66f83 docs(changelog): reorder ST drivers scopes]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I2c9f872e05fd8386e382b3a66a6f877ae9bbe282
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/294079
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Lionel DEBIEVE <lionel.debieve@foss.st.com>
---
 changelog.yaml | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/changelog.yaml b/changelog.yaml
index 20417b15a..1b8f193f7 100644
--- a/changelog.yaml
+++ b/changelog.yaml
@@ -1141,12 +1141,21 @@ subsections:
           - title: GPIO
             scope: st-gpio
 
+          - title: Regulator
+            scope: st-regulator
+
+          - title: Reset
+            scope: st-reset
+
           - title: SDMMC2
             scope: st-sdmmc2
 
             deprecated:
               - stm32_sdmmc2
 
+          - title: SPI
+            scope: st-spi
+
           - title: ST PMIC
             scope: st-pmic
 
@@ -1156,15 +1165,6 @@ subsections:
           - title: STPMIC1
             scope: stpmic1
 
-          - title: Regulator
-            scope: st-regulator
-
-          - title: Reset
-            scope: st-reset
-
-          - title: SPI
-            scope: st-spi
-
           - title: UART
             scope: st-uart
 
-- 
2.47.2

